src/main/kotlin/com/gatecheck/gatecheck/dao/request/RequestDataAccessService.kt

```diff
@@ -0,0 +1,20 @@
+package com.gatecheck.gatecheck.dao.request
+
+import com.gatecheck.gatecheck.model.Request
+import com.gatecheck.gatecheck.repository.RequestRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Repository
+import java.util.*
+
+
+@Repository("requestDAO")
```

Comment:
Consistency throughout a code database is very important, in the UserDataAccessService we've used @Repository("userDao"), notice that it's "Dao" and not "DAO" not a biggie but consistency is key 👌